docs/Date/index.md

```diff
@@ -0,0 +1,36 @@
+# Date
+A simple utility class for calculating time differences and rudimentry outputting.
+
+## Comparision
+
+Two date objects can be compared.
+
+### Example
+
+```csharp
+CF_Date dateA = CF_Date.CreateDateTime(2021, 12, 2, 10, 24, 3);
```

Comment:
Put a comment next to it with the datetime written out, so that people who are used to the freedom format, do not get scared.